test/application/onerror.js

```diff
@@ -48,7 +48,7 @@ describe('app.onerror(err)', () => {
 
     const output = stderr.inspectSync(() => app.onerror(err));
 
-    output.should.eql(['\n', '  Foo\n', '\n']);
+    output.should.eql(['\n  Foo\n\n']);
```

Comment:
@juliangruber Yes, with the change above, this change will be need. But if without that, this will fail the test.